test/testkit/matchers/prometheus/prommetric_utils.go

```diff
@@ -0,0 +1,68 @@
+package prometheus
+
+import prommodel "github.com/prometheus/client_model/go"
+
+// FlatMetricFamily holds all necessary information about a prometheus MetricFamily.
+// Gomega doesn't handle deeply nested data structure very well and generates large, unreadable diffs when paired with
+// the deeply nested structure of pmetrics.
```

Comment:
A bit of copy-paste here 😄 It has nothing to do with pmetrics

Reply:
Missed that one 😅